dot/peerset/peerstate.go

```diff
@@ -246,6 +246,8 @@ func (ps *PeersState) addNoSlotNode(idx int, peerID peer.ID) {
 	ps.sets[idx].noSlotNodes[peerID] = struct{}{}
 	n, err := ps.getNode(peerID)
 	if err != nil {
+		// TODO: Return the error
+		logger.Warnf("could not get node, error: %s", err)
```

Comment:
nit no need to mention it's an `error`, it should be obvious from the context preceding it

```suggestion
		logger.Warnf("could not get node: %s", err)
```